vector/src/main/java/im/vector/app/features/location/live/map/LiveLocationMapViewFragment.kt

```diff
@@ -71,11 +71,13 @@ class LiveLocationMapViewFragment @Inject constructor() : VectorBaseFragment<Fra
     private val viewModel: LiveLocationMapViewModel by fragmentViewModel()
 
     private var mapboxMap: WeakReference<MapboxMap>? = null
+    private var mapView: MapView? = null
```

Comment:
Why don't you use WeakReference for this one?

Reply:
In fact I am not a big fan of `WeakReference`, I prefer to explicitely dereferencing the value when needed. Like in this case, I reset the view to null in `onDestroyView`. Do you think I should switch to a `WeakReference`?

Reply:
I don't have a big opinion on this, I was asking because I observed the use of the `WeakReference` in the other fragments